src/pkg/mcp/tools/default_tool_cli.go

```diff
@@ -103,28 +91,28 @@ func (c *DefaultToolCLI) CaptureTermOutput(mode defangv1.DeploymentMode, estimat
 	return stdout.String()
 }
 
-func (c *DefaultToolCLI) LoadProject(ctx context.Context, loader cliClient.Loader) (*compose.Project, error) {
+func (DefaultToolCLI) LoadProject(ctx context.Context, loader cliClient.Loader) (*compose.Project, error) {
 	return loader.LoadProject(ctx)
 }
 
-func (c *DefaultToolCLI) CreatePlaygroundProvider(client *cliClient.GrpcClient) cliClient.Provider {
+func (DefaultToolCLI) CreatePlaygroundProvider(client *cliClient.GrpcClient) cliClient.Provider {
 	return &cliClient.PlaygroundProvider{FabricClient: client}
 }
 
-func (c *DefaultToolCLI) NewProvider(ctx context.Context, providerId cliClient.ProviderID, client *cliClient.GrpcClient) (cliClient.Provider, error) {
-	return c.NewProviderGrpc(ctx, providerId, client)
+func (DefaultToolCLI) NewProvider(ctx context.Context, providerId cliClient.ProviderID, client cliClient.FabricClient) (cliClient.Provider, error) {
+	return cli.NewProvider(ctx, providerId, client)
 }
 
-func (c *DefaultToolCLI) GetRegion(providerId cliClient.ProviderID) string {
+func (DefaultToolCLI) GetRegion(providerId cliClient.ProviderID) string {
 	return cliClient.GetRegion(providerId)
 }
 
-func (c *DefaultToolCLI) OpenBrowser(url string) error {
+func (DefaultToolCLI) OpenBrowser(url string) error {
 	// No-op stub implementation
 	return nil
```

Comment:
I don't get this. Isn't this supposed to be the actual implementation, not the one for testing?